tests/cache/tests.py

```diff
@@ -1833,16 +1833,13 @@ def test_cache_key_no_i18n(self):
     @override_settings(USE_I18N=False, USE_L10N=False, USE_TZ=True)
     def test_cache_key_with_non_ascii_tzname(self):
         # Regression test for #17476
-        class CustomTzName(timezone.UTC):
-            name = ''
-
-            def tzname(self, dt):
-                return self.name
+        class CustomTzName(timezone.utc):
```

Comment:
That's weird -- `timezone.utc` is a `tzinfo` instance, not a subclass, like `timezone.UTC` was.